Externals.cfg

```diff
@@ -12,7 +12,7 @@ protocol = git
 repo_url = https://github.com/ufs-community/ufs-weather-model
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = 3610672
+hash = de1546f
 local_path = sorc/ufs-weather-model
```

Comment:
@bbakernoaa, I changed the structure of the 'Externals.cfg' files to integrate online-cmaq into the develop branch. The basic concept is that we maintain a separate Externals.cfg file named 'Externals_AQM.cfg' for online-cmaq and we don't change any hashes in 'Externals.cfg'. Therefore, please change this hash to the original and change the hash of the ufs-weather-model in `Externals_AQM.cfg`. I think you didn't merge the latest online-cmaq branch into your branch yet becasue I moved `Externals_AQM.cfg` from 'externals' to the parent directory.